guides/doc-Planning_for_Project/topics/Deployment_Considerations.adoc

```diff
@@ -121,7 +121,7 @@ For distributing content from custom sources you need to create products and rep
 Red{nbsp}Hat repositories are signed with GPG keys by default, and it is recommended to create GPG keys also for your custom repositories.
 The configuration of custom repositories depends on the type of content they hold (RPM packages, or Docker images).
```

Comment:
```suggestion
(RPM packages or Docker images)
```

Reply:
On second thought, I'm not sure you need this sentence because you go on to explain the difference in the next sentence.

Reply:
I agree, this does not add any value. Dropped.